### PR TITLE
feat: search-overlay page

### PR DESCRIPTION
**Pull request by federicoisepponfincons**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

- added the search-overlay page

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): added page for specific search overlay usage

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by kyubisation** [comment]
I checked with @mcilurzo and he had the following feedback:

- The content (less padding) is drawn across the entire width of Ultra + instead of just on the grid
- Distance above to the input is not correct
- Close button (In the input) is also displayed when the input is empty.
- Close button (in the input) doesn't seem to work
- Title "often searched" is missing
- "Show all 13 results" button is missing
- Striking, but I wouldn't correct it: The border thickness of the magnifying glass icon is thicker in Florian's designs (like in the header).

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Also approved by Florian
Thank you for your effort 😃

**Comment by kyubisation** [comment]
@federicoisepponfincons Can this be merged?

**Comment by kyubisation** [comment]
Pending discussion with @mcilurzo

File: src/storybook/pages/search-overlay/search-overlay-instant-results.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                data-icon-small=""
+                role="button"
+                variant="primary"
+                size="l"
+              ></sbb-button>
+            </sbb-form-field>
+            <div class="search-result__wrapper">
+              <a href="http://www.sbb.ch/" class="search-result-item">
+                <sbb-title visual-level="6" negative>
+                  Result title 1<span class="subtitle">Lorem ipsum subtitle</span>
```
Comment (kyubisation) [inline src/storybook/pages/search-overlay/search-overlay-instant-results.stories.tsx:109]: I missed this initially. I think the subtitle should not be nested in the sbb-title, but instead be maybe a `<p>` below?
What do you think?

Comment (federicoisepponfincons) [reply src/storybook/pages/search-overlay/search-overlay-instant-results.stories.tsx]: I think it's better in a separate <p> element, updated it

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
Focus styles for results and also for the input field are not correct / visible

**Comment by jeripeierSBB** [comment]
I fixed the focus outline for the results

**Pull request closed without merging**